chef/cookbooks/keystone/recipes/ha.rb

```diff
@@ -65,5 +65,12 @@
     only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
   end
 
+  crowbar_pacemaker_order_only_existing "o-#{service_name}" do
+    ordering "postgresql rabbitmq cl-#{service_name}"
+    score "Mandatory"
```

Comment:
I'm thinking it should be `Optional`, not `Mandatory` as this is really an issue on startup. Keystone is able to recover from a db that fails after startup.